test/test_psutil.py

```diff
@@ -2475,6 +2475,55 @@ def test_Popen(self):
             proc.wait()
             self.assertIsNotNone(proc.returncode)
 
+    @unittest.skipUnless(hasattr(psutil.Process, "environ"),
+                         "environ not available")
+    def test_environ(self):
+        self.maxDiff = None
+        p = psutil.Process()
+        d = p.environ()
+        d2 = os.environ.copy()
+
+        if OSX:
+            for key in (
+                "__CF_USER_TEXT_ENCODING",
+                "VERSIONER_PYTHON_PREFER_32_BIT",
+                "VERSIONER_PYTHON_VERSION",
+            ):
+                d.pop(key, None)
+                d2.pop(key, None)
+
+        self.assertEqual(d, d2)
+
+    @unittest.skipUnless(hasattr(psutil.Process, "environ"),
+                         "environ not available")
+    @unittest.skipUnless(POSIX, "posix only")
+    def test_weird_environ(self):
```

Comment:
perhaps add a comment explaining what you're testing exactly (aka why it is "weird"?).